netlify.toml

```diff
@@ -8,7 +8,7 @@ command = "git submodule update --init --recursive --depth 1 && make non-product
 
 [build.environment]
 NODE_VERSION = "10.20.0"
-HUGO_VERSION = "0.111.3"
+HUGO_VERSION = "0.111.9"
```

Comment:
Don't bump the Hugo version in this PR.

Reply:
Yup, this was accidentally included. I'm reverting and will repush.